src/components/BRB/BRBPartners.tsx

```diff
@@ -95,6 +95,7 @@ const PartnersContainer = styled.div`
 const Header = styled.h3`
   font-size: 46px;
   font-weight: 400;
+  font-family: Glancyr;
```

Comment:
Fallback font should be mentioned: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family